README.md

```diff
@@ -27,7 +27,57 @@ These are the primary software versions used in production, and you should be ab
   - `brew doctor`
 
 
-### **2. Python**  
+### **2. (Mac) Install or update other dependencies:**
+
+```bash
+brew install libevent libmagic libxml2 libxslt openssl graphviz nginx
+brew install freetype libjpeg libtiff littlecms webp chromedriver
+brew tap petere/postgresql
+brew install postgresql@9.3
+brew link --force postgresql@9.3
+brew install --force node@6
+brew link node@6 --force
+brew cask install java8
+brew tap garrow/homebrew-elasticsearch17
+brew install elasticsearch@1.7
```

Comment:
Do we need this line and the one above it?  They are es 1.7

Reply:
I agree. I will remove the es1.7 instructions.